Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up
+
+import random
+
+#These are my constants for the future I will UPPER_SNAKE_CASE
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 'Nine':9, 'Ten':10, 'Jack':10,
+         'Queen':10, 'King':10, 'Ace':11}
+
+if __name__=="__main__":
+
+    class Card():
+
+        def __init__(self, suit, rank):
+            self.suit = suit
+            self.rank = rank
+            self.value = values[rank]
+
+        def __str__(self):
+            return f"{self.rank} of {self.suit}"
+
+
+    class Deck():
+
+        def __init__(self):
+            self.all_cards = []
+            for suits_card in suits:
+                for rank_card in ranks:
+                    self.all_cards.append(Card(suits_card, rank_card))
+
+        def shuffle(self):
+            random.shuffle(self.all_cards)
+
+        def deal(self):
+            return self.all_cards.pop()
+
+        def __str__(self):
+            for x in range(0, len(self.all_cards)):
+                print(f"{self.all_cards[x]}")
+            return f"This is the current cards in the deck"
+
+
+    class Hand():
+
+        def __init__(self):
+            self.all_cards = []
+            # player overall value towards 21 = winning! Blackjack
+            self.player_value = 0
+            # counter to keep track of when an ace is in player hand
+            self.ace_counter = 0
+
+        # adding a card to player hand, calculating value towards 21 & Ace counter
+        def add_card(self, new_card):
+            self.all_cards.append(new_card)
+            self.player_value += new_card.value
+            if new_card.rank == "Ace":
+                self.ace_counter += 1
+
+        # Best way to change the Ace value is by subtracting from the overall value sum
+        def adjust_ace(self):
+            while self.player_value > 21 and self.ace_counter:
+                self.player_value -= 10
+                self.ace_counter -= 1
+
+        def __str__(self):
+            # for x in range(0,len(self.all_cards)):
+            # print(f"{self.all_cards[x]}")
+            return f"Hand Value is {self.player_value}"
+
+
+    class Chips():

```

Comment:
Seems like this should be name as `Player` class. Haven't finished reading the rest though.